modules/CmpdReg/src/client/src/Parent.js

```diff
@@ -156,7 +156,7 @@ $(function() {
 	            var optionToInsert = null;
             }
             this.stereoCategoryCodeController =
-                this.setupCodeController('stereoCategoryCode', 'stereoCategorys', 'stereoCategory', optionToInsert);
+                this.setupCodeController('stereoCategoryCode', 'stereoCategories', 'stereoCategory', optionToInsert);
```

Comment:
Ended up with a different name here because we already had a roo `api/v1/stereoCategories` route.